Dockerfile

```diff
@@ -4,14 +4,15 @@ RUN mkdir /app
 WORKDIR /app
 ARG VERSION
 
-ADD . /app/
+COPY . ./
 RUN go build -ldflags=-s -ldflags=-w -ldflags=-X=github.com/miguelmota/cointop/cointop.version=$VERSION -o main .
-RUN git clone https://github.com/cointop-sh/colors && rm -Rf colors/.git*
+ADD https://github.com/cointop-sh/colors/archive/master.tar.gz ./
+RUN tar zxf master.tar.gz --exclude images
```

Comment:
Although this change creates an additional layer I think it is better as it fetches less data.

In case a clone is indeed preferred I'd like to suggest adding `--depth 1` to the clone command, i.e.:
`RUN git clone --depth 1 https://github.com/cointop-sh/colors && rm -Rf colors/.git*`